app/src/pages/OnDeviceDisplay/ProtocolDetails/index.tsx

```diff
@@ -98,25 +145,50 @@ const ProtocolSectionTabs = (props: ProtocolSectionTabsProps): JSX.Element => {
 const Summary = (props: {
   author: string | null
   description: string | null
+  date: string | null
 }): JSX.Element => {
   const { t } = useTranslation('protocol_details')
   return (
     <Flex flexDirection={DIRECTION_COLUMN} gridGap={SPACING.spacing2}>
-      <Flex gridGap={SPACING.spacing2}>
-        <StyledText
-          as="h2"
-          textTransform={TYPOGRAPHY.textTransformCapitalize}
-        >{`${t('author')}: `}</StyledText>
+      <Flex
+        fontSize={TYPOGRAPHY.fontSize22}
+        fontWeight={TYPOGRAPHY.fontWeightSemiBold}
+        gridGap={SPACING.spacing2}
+        lineHeight={TYPOGRAPHY.lineHeight28}
+      >
+        <StyledText textTransform={TYPOGRAPHY.textTransformCapitalize}>{`${t(
+          'author'
+        )}: `}</StyledText>
         <StyledText>{props.author}</StyledText>
       </Flex>
-      <StyledText>{props.description}</StyledText>
+      <StyledText
+        fontSize={TYPOGRAPHY.fontSize22}
+        fontWeight={TYPOGRAPHY.fontWeightRegular}
+        lineHeight={TYPOGRAPHY.lineHeight28}
+      >
+        {props.description}
```

Comment:
Can we destruct props?
const { author, description, data } = props 
A few components do `props.something` but we avoid that.

```suggestion
        {description}
```